src/swell/tasks/link_geos_output.py

```diff
@@ -80,5 +95,53 @@ def link_restart(self):
 
         return src, dst
 
+    # ----------------------------------------------------------------------------------------------
+
+    def prepare_cice6(self):
```

Comment:
Type hint (this syntax should be supported in Python 3.9+; if we need 3.8 or lower, we will need to do `from typing import Tuple` above and, here, `Tuple(int, int)`.

```suggestion
    def prepare_cice6(self) -> tuple[str, str]:
```

_Strictly_ speaking, it might be good to take advantage of Python 3's built-in `pathlib` for paths --- then, this would be more clearly `tuple[Path, Path]`, and we would be able to replace a bunch of `os.path.join(somepath, someotherpath)`, `os.path.exists(somepath)` with syntactic sugar like `somepath / someotherpath` (to join paths) and `somepath.exists()`. But I'm not going to make you do that here 😉  --- just flagging as food for thought for the future.

Reply:
Sounds good!